### PR TITLE
add crescendo support

### LINKED ISSUE #1350: Add Preview Network
Add the crescendo network to anywhere it needs to be. For example, `flow.json` needs access node setup and anywhere in the code where something like mainnet or testnet is supported should also include crescendo: https://github.com/search?q=repo%3Aonflow%2Fflow-cli+Testnet&type=code

### PR DESCRIPTION
**Pull request by ianthpun**
Closes #1350 

## Description

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: internal/transactions/build.go [hunk 1]
```diff
@@ -149,4 +149,5 @@ func parseAddress(value string) (flowsdk.Address, bool) {
 	return address, address.IsValid(flowsdk.Mainnet) ||
 		address.IsValid(flowsdk.Testnet) ||
 		address.IsValid(flowsdk.Emulator)
+	//|| address.IsValid(flowsdk.Crescendo) //TODO: add crescendo, it is missing in flowsdk
```
Comment (ianthpun) [inline internal/transactions/build.go:152]: need this merged first https://github.com/onflow/flow-go-sdk/pull/558

File: internal/util/util.go [hunk 2]
```diff
@@ -72,6 +72,7 @@ func GetAddressNetwork(address flowsdk.Address) (flowsdk.ChainID, error) {
 		flowsdk.Mainnet,
 		flowsdk.Testnet,
 		flowsdk.Emulator,
+		// flowsdk.Crescendo TODO: add crescendo
```
Comment (ianthpun) [inline internal/util/util.go:75]: need this merged first https://github.com/onflow/flow-go-sdk/pull/558

File: internal/accounts/staking-info.go [hunk 3]
```diff
@@ -156,6 +156,7 @@ func envFromNetwork(network flowsdk.ChainID) tmpl.Environment {
 		}
 	}
 
+	// TODO: maybe add cresecendo here?
```
Comment (bjartek) [inline internal/accounts/staking-info.go:159]: crescendo addresses will be the same as testnet here.

**Review by bjartek — CHANGES_REQUESTED** [review]

**Comment by ianthpun** [comment]
closing as we aren't using crescendonet anymore

**Pull request closed without merging**